pages/blog/whats-new-in-graphql-cli-4.1.0.mdx

```diff
@@ -0,0 +1,65 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: "What's new in GraphQL CLI 4.1.0",
+  author: 'ephelan',
+  tags: ['react'],
+  date: '2020-10-12',
+  description: "An overview of what we've added in this release of GraphQL CLI",
+  image:
+    'https://user-images.githubusercontent.com/20847995/67651234-85bf1500-f916-11e9-90e5-cb3bd0e6a338.png',
+};
+
+export default withArticle({ ...meta });
+
+[GraphQL CLI](http://graphql-cli.com) is your one-stop shop for developing full-stack GraphQL applications in Node.js. Since the release of 4.0.0, we have added a number of enhancements which will further elevate your GraphQL development experience.
+
+## GraphQL Code Generator
+
+[Add what is new in GraphQL Codegen]
+
+To use GraphQL Code Generator from the CLI, run `graphql codegen` from your application root:
+
+```shell
+$ graphql codegen
+```
+
+Check out the [codegen command docs](https://graphql-cli.com/codegen) on our new website for configuration and installation instructions.
+
+## GraphQL Config
+
+[Add what is new in GraphQL Config]
```

Comment:
Not sure if you want anything added here.

Reply:
@kamilkisiela Do you want to add anything here?